### PR TITLE
Responsive tables

### LINKED ISSUE #141: Convert tables to list view
Convert all table layouts to list view. See [mockup](https://projects.invisionapp.com/d/main/default/#/console/20707215/443036071/preview).

> [LATER] At small screen breakpoint, implement stacked designs for table rows instead of h-scroll

> At small screen breakpoint, make sure all main content within viewport and tables support horizontal scroll.

### PR DESCRIPTION
**Pull request by erinesullivan**
# Overview
Tables are not mobile friendly by default. This PR makes it so! See [mockup](https://projects.invisionapp.com/d/main/default/#/console/20707215/443036071/preview).

Resolves #141 

## Testing
* Check out all of the pages and play around with the browser window size. Make sure nothing is borked.

### CODE REVIEW

File: views/requests.erb [hunk 1]
```diff
@@ -31,12 +32,14 @@
           <% end %>
         </td>
         <td>
+          <span aria-hidden="true">Requested:</span>
           <%= hold_request.request_date %>
         </td>
         <td>
+          <span aria-hidden="true">Expires:</span>
           <%= hold_request.request_date %>
```
Comment (erinesullivan) [inline views/requests.erb]: @niquerio `Line 36` and `Line 40` are pulling in the same method. Is this intentional?

File: views/requests.erb [hunk 2]
```diff
@@ -130,12 +136,14 @@
             <% end %>
           </td>
           <td>
+            <span aria-hidden="true">Pickup date:</span>
             <%= booking_request.booking_date %>
           </td>
           <td>
+            <span aria-hidden="true">Expires:</span>
             <%= booking_request.booking_date %>
```
Comment (erinesullivan) [inline views/requests.erb]: @niquerio `Line 140` and `Line 144` are pulling in the same method. Is this intentional?

File: views/requests.erb (hunk 1 above)
Comment (niquerio) [reply views/requests.erb]: nope, not intentional. I added expiry.

File: views/requests.erb (hunk 2 above)
Comment (niquerio) [reply views/requests.erb]: same deal as above

**Review by niquerio — COMMENTED** [review]
fixed!

**Pull request merged**